wcivf/templates/base.html

```diff
@@ -49,6 +49,26 @@
     {% endif %}
 {% endblock extra_site_css %}
 
+{% block top_banner %}
+<main>
```

Comment:
Our primary content is already inside a `<main>`. You're only supposed to have one `<main>` element in a document.

Reply:
Having looked at it in a bit more detail (I'm just looking at nicking the same thing for WDIV), I can see why you've done this. My proposed solution would be to fix this in the theme: https://github.com/DemocracyClub/dc_base_theme/pull/52 tag a release and bump the theme in WDIV/WCIVF rather than have 2 `<main>`s

Reply:
Great – I'll update the theme when that's in ad bumped

Reply:
:+1:  `0.3.11` is tagged